### PR TITLE
Master fixes

### PR DESCRIPTION
**Pull request by alexcouper**
Fixes that make this actually build and release properly.

See https://circleci.com/gh/ployst/component-builder/28 for a working build that resulted in release of this library to pypi.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 setup(
     name='component_builder',
     description='Build runner of components for testing/releasing purposes.',
-    version='0.1',
+    version=version,
```
Comment (txels) [inline setup.py:36]: LOL yeah, nice bug.

File: src/component_builder/release.py [hunk 2]
```diff
@@ -10,13 +10,11 @@ def run(components):
         if not component.release_process:
             print("{0}: doesn't support releasing".format(component.title))
             continue
-        env = os.environ.copy()
-        env.update(component.env)
         if component.release_process == 'docker':
+            env = os.environ.copy()
+            env.update(component.env)
```
Comment (txels) [inline src/component_builder/release.py:15]: Is this docker-release-specific?

**Comment by txels** [comment]
Circle build log exposes ployst-bot credentials.

**Comment by alexcouper** [comment]
@txels sorted

**Review by txels — APPROVED** [review]
MoG

**Pull request merged**